### ISSUE TITLE
Errors 202, 206 and 209 triggered inside raw blocks {% raw %}/{% endraw %}

### ISSUE THREAD

**Issue by Lomanic** [warpnet/salt-lint#201]
**Describe the bug**
While implementing CI using salt-lint on a Salt repo, I stumbled upon the following behavior, where errors were reported even inside raw blocks delimited by `{% raw %}`/`{% endraw %}`.

**To Reproduce**
```bash
cat > test.sls <<EOF
environments:
  variables:
    heketi:
      {% raw %}
      HEKETI_CLI_SERVER: \$(kubectl --kubeconfig=/etc/kubernetes/admin.conf get svc/heketi -n glusterfs --template "http://{{.spec.clusterIP}}:{{(index .spec.ports 0).port}}" 2>/dev/null || echo 'Go root!')
      {% endraw %}
EOF
docker run --rm -v "$PWD":/data:ro -it warpnetbv/salt-lint:0.4.2 -x 204 test.sls
```
Result
```
[206] Jinja variables should have spaces before and after: {{ var_name }}
test.sls:5
      HEKETI_CLI_SERVER: $(kubectl --kubeconfig=/etc/kubernetes/admin.conf get svc/heketi -n glusterfs --template "http://{{.spec.clusterIP}}:{{(index .spec.ports 0).port}}" 2>/dev/null || echo 'Go root!')

```

**Expected behavior**
Errors shouldn't be reported inside `raw` elements, or at least the ones about Jinja formatting (202, 206 and 209) should be ignored. Inside a raw block, we are not evaluating Jinja markup, so we don't care if some `{{` is not properly spaced (in fact, the raw block is meant to prevent errors during Jinja rendering by Salt as it would be interpreted as an undefined variable).

**Desktop:**
 - OS: Debian 9 amd64
 - Version 0.4.2

**Comment by roaldnefs** [comment]
Thanks @Lomanic for your bug report! I've just confirmed this is indeed an issue.

One possible solution would be to filter all Jinja escaping using a regex similar to: `(\{%\sraw\s%\}(?:\n?.*?)+\{%\sendraw\s%\})`.

**Comment by Lomanic** [comment]
Thanks for taking time to fix that @roaldnefs :+1:

**Comment by roaldnefs** [comment]
I'm happy to help, @Lomanic!
